common/tls/src/main/java/io/helidon/common/tls/spi/TlsManagerProvider.java

```diff
@@ -24,4 +28,22 @@
  */
 public interface TlsManagerProvider extends ConfiguredProvider<TlsManager> {
 
+    /**
+     * Provides the ability to have a unique {@link TlsManager} per unique {@link Config} instance provided.
+     *
+     * @param config     the raw config instance
+     * @param configBean the config bean instance
+     * @param name       the config bean instance name
+     * @param creator    the creator to apply if not already in cache, which takes the config bean instance
+     * @return the tls manager instance from cache, defaulting to creation from the {@code creator} if not in cache
+     */
+    static TlsManager getOrCreate(Config config,
```

Comment:
why is there a `name` and  `config` parameter, when theser are not used in the code? Please remove them
Also update the signature to what I wrote above

Reply:
I was thinking we might need them for future use.  I'll remove them but at least consider that.

Reply:
pushed